vulture/lines.py

```diff
@@ -23,27 +25,26 @@
     })
 
 
-# Reinventing wheel here to keep vulture lightweight.
-def memoize(func):
-    _cache = {}
+def _get_range(entry):
+    if isinstance(entry, ENTRY_POINTS):
+        last_body = entry.body[-1]
+        while isinstance(last_body, tuple(TRAVERSABLE_FIELDS.keys())):
+            fields = TRAVERSABLE_FIELDS[last_body.__class__]
+            for field in reversed(fields):
+                children = getattr(last_body, field)
+                if children:
+                    break
+            last_body = children[-1]  # The last child.
```

Comment:
Probably clearer:

    if children:
        last_body = children[-1]
        break

The comment is not needed.